src/web/export-file.jsp

```diff
@@ -1,4 +1,3 @@
-<%@ page contentType="text/html; charset=UTF-8" %>
```

Comment:
I think I already addressed this in https://github.com/igniterealtime/openfire-userImportExport-plugin/commit/c8e2ebbc0813cac2a50c998930868a293399010f